SwiftPackage/Sources/BridgeClientExtension/File Uploading/V1/BridgeFileUploadManager.swift

```diff
@@ -499,18 +487,23 @@ public class BridgeFileUploadManager: NSObject, URLSessionBackgroundDelegate {
     let retryUploadsKey = "RetryUploadsKey"
     
     /// A mapping for implementation-specific details for each registered Bridge upload API.
-    var bridgeFileUploadApis = [String : BridgeFileUploadAPI]()
+    lazy private var bridgeFileUploadApis: [String : BridgeFileUploadAPI] = [
+        participantFileUploadAPI.apiString : participantFileUploadAPI,
+        studyDataUploadAPI.apiString : studyDataUploadAPI
+    ]
     
     /// The minimum delay before retrying a failed upload (in seconds).
     var delayForRetry: TimeInterval = 5 * 60
     
     /// BridgeFileUploadManager uses the BackgroundNetworkManager singleton to manage its background URLSession tasks.
```

Comment:
```suggestion
    /// BridgeFileUploadManager uses a BackgroundNetworkManager instance to manage its background URLSession tasks.
```
(since it's not a singleton anymore?)